installer/scripts/omsadmin.sh

```diff
@@ -95,6 +95,7 @@ PROC_LIMIT_CONF=/etc/security/limits.conf
 # SCOM variables
 SCX_SSL_CONFIG=/opt/microsoft/scx/bin/tools/scxsslconfig
 OMI_CONF_FILE=/etc/opt/omi/conf/omiserver.conf
+OMI_CONF_EDITOR=/opt/omi/bin/omiconfigeditor
```

Comment:
@jeffaco Is there a minimum version of OMI that supports this? If this was only added in a recent version, then we may want to update our dependencies: https://github.com/Microsoft/OMS-Agent-for-Linux/blob/master/installer/datafiles/linux_rpm.data#L10 and https://github.com/Microsoft/OMS-Agent-for-Linux/blob/master/installer/datafiles/linux_dpkg.data#L9